src/nvidia-modeset/kapi/src/nvkms-kapi.c

```diff
@@ -2432,6 +2435,18 @@ static NvBool NvKmsKapiPrimaryLayerConfigToKms(
         changed = TRUE;
     }
 
+    if (layerRequestedConfig->flags.dstXYChanged || bFromKmsSetMode) {
+        /* If the main layer doesn't support window mode,
```

Comment:
nit: Please align the `*`'s in block comments and have the comment close on a newline like so:
```
/*
 * If the main layer...
 * ...
 */
```